src/components/ArticleCard.jsx

```diff
@@ -1,5 +1,26 @@
-import React from "react";
+import { Image } from "react-bootstrap";
 
-export default function ArticleCard() {
-  return <div></div>;
+export default function ArticleCard({
+  author,
+  title,
+  article_id,
+  topic,
+  created_at,
+  votes,
+  article_img_url,
+  comment_count,
+}) {
+  return (
+    <div>
+      <Image
+        style={{ display: "block", width: "900%", marginLeft: "auto", marginRight: "auto" }}
+        src={article_img_url}
+        alt="an image to describe each articles"
+        fluid
+      />
+      <br />
+      <h6>{title}</h6>
+      <hr/>
+    </div>
```

Comment:
It would be good to include a bit more information about the article on these article cards 
At a minimum I would include information that you're going to be sorting by later, such as the date it's created, author, number of comments etc.

Reply:
Done thanks :)